samples/sensor/thermometer/sample.yaml

```diff
@@ -10,3 +10,5 @@ tests:
       - frdm_k22f             # tcn75a
       - robokit1              # ntc_thermistor
       - adi_eval_adin1110ebz  # adt7420
+      - frdm_mcxn947/mcxn947/cpu0 # p3t1755
+      - frdm_mcxn947/mcxn947/cpu0/qspi # p3t1755
```

Comment:
Pick one. This list of boards tries to cover different sensor drivers, not different boards.

Reply:
I have made the change.